client/packages/system/src/Stock/DetailView/Footer.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { FC } from 'react';
+import {
+  Box,
+  useTranslation,
+  AppFooterPortal,
+  DialogButton,
+  LoadingButton,
+} from '@openmsupply-client/common';
+import { FormInputData } from '@openmsupply-client/programs';
+
+interface FooterProps {
+  isSaving: boolean;
+  isDirty?: boolean;
+  validationError?: string | boolean;
+  inputData?: FormInputData;
+  showSaveConfirmation: () => void;
+  showCancelConfirmation: () => void;
+}
+
+export const Footer: FC<FooterProps> = ({
+  isSaving,
+  isDirty,
+  inputData,
+  showSaveConfirmation,
+  showCancelConfirmation,
+}) => {
+  const t = useTranslation();
+
+  return (
+    <AppFooterPortal
+      Content={
+        <Box
+          gap={2}
+          display="flex"
+          flexDirection="row"
+          alignItems="center"
+          height={64}
+        >
+          <Box
+            flex={1}
+            display="flex"
+            justifyContent="flex-end"
+            gap={2}
+            marginLeft="auto"
+          >
+            {isDirty && (
+              <DialogButton
+                variant="cancel"
+                disabled={isSaving}
+                onClick={() => showCancelConfirmation()}
+              />
+            )}
```

Comment:
Can this be:

```suggestion
              <DialogButton
                variant="cancel"
                disabled={!isDirty || isSaving}
                onClick={() => showCancelConfirmation()}
              />
```
The save button disabling/enabling, but the cancel button appearing/disappearing is weird. Usual design has been disable everywhere rather than hide?